src/add_lable.py

```diff
@@ -0,0 +1,58 @@
+import pandas as pd
+from dotmap import DotMap
+from github import Github
+from github import Auth
+from github.Repository import Repository
+
+ORIGINAL_ISSUES_FILE = "../res/issues.json.zip"  # ../res/ o res/
```

Comment:
If possible avoid global variable, pass the file as an argument to the script.
Here some [examples](https://docs.python.org/3/library/argparse.html).

Reply:
Reopened.